wgengine/router/router_linux.go

```diff
@@ -1065,7 +1065,7 @@ func checkIPv6() error {
 	}
 
 	if err := checkIPRuleSupportsV6(); err != nil {
```

Comment:
move this before the ip6tables check so the two policy routing/rule checks are next to each other?